tools/hicup/hicup_deduplicator.xml

```diff
@@ -0,0 +1,43 @@
+<tool id="hicup_deduplicator" name="Deduplicator" version="0.1.0">
+    <requirements>    
+        <requirement type="package" version="0.5.9">hicup</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" />
+    </stdio>
+    <command><![CDATA[
+        hicup_deduplicator $input_file
```

Comment:
All files (paths) and all text parameters needs to be quoted with single quotes.